Spelunker: Implement New Game (#3282)
ArchipelagoMW/Archipelago: worlds/spelunker/docs/en_Spelunker.md

# Spelunker

## Where is the options page?

The [player options page for this game](../player-options) contains all the options you need to configure and export a config file.

## What does randomization do to this game?

All of the collectable items scattered throughout the cave are randomized into the Multiworld pool, with the exception of Energy refills.

## What is the goal of Spelunker when randomized?

The goal is to reach the bottom of the cave and discover the Pyramid of Gold.

> ## What items and locations get shuffled?
> 
> Locations consist of any collectable items aside from Energy refills. Optionally, the player may also randomize the hidden items.

Reviewer: ```suggestion
## What items and locations get shuffled?

The shuffled items and locations are the game's collectible items aside from Energy refills. Optionally, the player may also randomize the hidden items.
```

I'm not super attached to the wording of this suggestion, though you definitely want to change "collect**a**ble" to "collect**i**ble"

## Which items can be in another player's world?

Any shuffled item can be in other players' worlds.

## What does another world's item look like in Spelunker

All items, whether they are for another world or not, appear as Archipelago symbols, with Progression items being colored blue and white, and all others being green and red.

## When the player receives an item, what happens?

> When the player receives an item, a fanfare or tune is played and the player receives the corresponding equipment or powerup.

Reviewer: Are there two different sound effects, one which you'd call a fanfare and one which you'd call a tune, or is there just one sound effect? If it's just one, would recommend using just fanfare (or just tune).

Author: Yes, most items only play a fanfare but a handful of power ups change the music for the duration of the powerup’s effect.
